roadservice/core/models.py

```diff
@@ -720,3 +725,26 @@ def delete_mission_type(self, mission_type):
             mission_type.delete()
         except ProtectedError:
             raise BusyResourceError()
+
+
+class SMSSender(KavenegarAPI):
+    __instance = None
+
+    @classmethod
+    def get_instance(cls):
+        if cls.__instance is None:
+            cls()
+        return cls.__instance
+
+    def __init__(self):
+        if SMSSender.__instance is not None:
+            raise SingletonInitError()
+        super().__init__(settings.SMS_API_KEY)
+        SMSSender.__instance = self
+
+    def sms_send_to_user(self, user, message):
```

Comment:
Change suggestion: `send_sms_to_user` or even `send_to_user`. (Remember that the usage is `sms_sender.send_sms_to_user()` so `sms` is kinda redundant)

Reply:
I see that your naming follows Kavengar's, but I believe it's not pretty :D

Reply:
And bear in mind, Kavengar's class name, doesn't suggest that it can only be used to send SMS. For example they can provide another method to send mass emails. Hence, 'sms' is not redundant in their method name, but ours is. However, I still believe they should've named it 'send_sms', not the other way around.